package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "appsflyer-capacitor-plugin",
-  "version": "6.8.2",
+  "version": "6.9.2-rc1",
```

Comment:
merging to develop with this version name? make sure you don't merge it later to master like this